README.md

```diff
@@ -45,6 +45,64 @@ $ git clone https://github.com/okfn-brasil/notebooks.git
 
 We use serenata toolbox to download or generate the datasets. To install and use the toolbox just checkout the [toolbox's guide](https://github.com/okfn-brasil/serenata-toolbox#serenata-de-amor-toolbox).
 
+### Deploy in production with docker
+
+#### Prerequisites
+
+[Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+[Docker Compose](https://docs.docker.com/compose/install/)
+[Dockerhub](https://hub.docker.com/), To create the dockerhub account to store the image
+
+> Note: Considering the dockerhub user name as  okbr, replace the dockerhub user name accordingly
+
+#### Prepare Docker Images
+
+- Create the docker images
+
+```bash
+docker build -t notebooks .
+```
+
+As we create the docker image, it will be stored in local docker images cache
+
+- Tag the image to maintain th version of the docker image
+
+```bash
+docker tag notebooks notebooks:1.0
```

Comment:
How about keeping 0.1 all across this document?

Reply:
No it is just a current release version, lets say in future if someone tryning to push latest all the time it will replcae the tag and image in the docker hub